### ISSUE TITLE
[WII] Many problems in version 1.9.0

### ISSUE THREAD

**Issue by Edu789** [libretro/RetroArch#11281]
The cores fbalpha2012_cps1, cps2, cps3, fbalpha2012_libretto and fbalpha2012_neogeo stopped working
Do not work atari800_libretro and 81_libretro
The core mame2000 many games do not work and others are working with graphical bugs
The cores mednafen_pcfx, quasi88 and x1 are very slow
In the core snes9x the satellaview is not working properly, the game starts normally and then crashes
The cores freeintv, mednafen_lynx, mednafen_ngp, mednafen_vb, quasi88, stella, stella2014 and theodore don't work as Wiiflow plugins
The cores gambatte, genesis_plus_gx, mednafen_pce_fast, mednafen_wswan, snes9x2010 stopped working with the roms zipped in wiiflow, when entering the game, it starts and already crashes, you have to turn the video game off and on again
The cores are not automatically recognizing the classic controller

**Comment by giovanifaganello** [comment]
Just updated my RetroArch on PSClassic (to 1.9.0) and had a problem with a MAME game. I am pretty sure it was working on 1.8.5. The title is ALCON, using MAME 2003-Xtreme. It loads the game fine, but I do get a game message ERROR. It is not a retroarch error or a MAME warning error. It is a error presented by the game itself...

**Comment by twinaphex** [comment]
We need more coders contributing for Wii. It's that simple.

**Comment by twinaphex** [comment]
As for WiiFlow, not our responsibility. If somebody wants to fix it and send a PR then fine, but we are only concerned about RetroArch as a standalone application.

**Comment by Cee123** [comment]
I already made an issue for this before

https://github.com/libretro/RetroArch/issues/11217

@Edu789 Do you get the issues with the controllers not responding as well?

**Comment by Edu789** [comment]
No

**Comment by Cee123** [comment]
> No

Ok...

**Comment by Edu789** [comment]
Yes, the Wii remote and the Gamecube control work normally, only the classic control that does not automatically recognize in cores...

**Comment by Cee123** [comment]
> 
> 
> Yes, the Wii remote and the Gamecube control work normally, only the classic control that does not automatically recognize in cores...

Should all be working in the latest nightlies now. The controller issues seem to be fixed.

**Comment by Edu789** [comment]
Yes, hopefully they fix the other problems I mentioned ...

**Comment by SPILAB** [comment]
Hello everyone, if that can help a little bit, I tested the nightbuild 2021-01-24_RetroArch, and that sound much better.
The cores fbalpha2012_cps1, cps2, fbalpha2012_libretto work for example. (I cannot find the cps3 anymore, maybe I doing something wrong). Tell me if I can do some tests for you.

**Comment by Cee123** [comment]
Just a couple of other cores working for wii
[prosystem_libretro_wii.zip](https://github.com/libretro/RetroArch/files/5915516/prosystem_libretro_wii.zip)
[reminiscence_libretro_wii.zip](https://github.com/libretro/RetroArch/files/5915517/reminiscence_libretro_wii.zip)
[smsplus_libretro_wii.zip](https://github.com/libretro/RetroArch/files/5915518/smsplus_libretro_wii.zip)
[xrick_libretro_wii.zip](https://github.com/libretro/RetroArch/files/5915519/xrick_libretro_wii.zip)

And of course DosBox is all working now (thanks to metallic77)
[dosbox_libretro_wii.zip](https://github.com/libretro/RetroArch/files/5915521/dosbox_libretro_wii.zip)

**Comment by Cee123** [comment]
The cores for CPS3 and FixGb and FixNes are not in the nightly anymore. But I still have these old versions.
[fbalpha2012_cps3_libretro_wii.zip](https://github.com/libretro/RetroArch/files/5915527/fbalpha2012_cps3_libretro_wii.zip)
[fixgb_libretro_wii.zip](https://github.com/libretro/RetroArch/files/5915528/fixgb_libretro_wii.zip)
[fixnes_libretro_wii.zip](https://github.com/libretro/RetroArch/files/5915529/fixnes_libretro_wii.zip)

**Comment by Cee123** [comment]
@SPILAB The CPS3 core used to work great. I've got an old version of it there, works well.

**Comment by IAmBoring** [comment]
Have you checked latest? Lots of 'missing' cores there.
http://buildbot.libretro.com/nightly/nintendo/wii/latest/

**Comment by Cee123** [comment]
> Have you checked latest? Lots of 'missing' cores there.
> http://buildbot.libretro.com/nightly/nintendo/wii/latest/

Oh right i had not checked that recently, no. I see they've added a bunch more. Last time i checked it, which was probably a few weeks ago, they were not there.

**Comment by Cee123** [comment]
Just tried the latest nightly, and the controls aren't responding with most of the cores except for in the RetroArch menu. Dosbox is gone. Vice freezes on a black screen. And CPS3 crash dumps.

**Comment by Cee123** [comment]
Update: Sorry my apologies. The only cores where the controls don't work seem to be the FB Alpha 2012 cores. All the rest I've tried work fine.

But I did have some issues with these cores
FB Alpha 2012 - Controls don't work
FB Alpha 2012 CPS1 - Controls don't work
FB Alpha 2012 CPS2 - Controls don't work
FB Alpha 2012 CPS3 - Crash Dumps
FB Alpha 2012 Neo Geo - Freezes at the Bios screen, controls probably don't work either
MGBA - Crash Dumps
Snes9x2002 - Works now, but quiet, crackling audio
SMS Plus - Crash Dumps
Vice cores - Freeze on a black screen

**Comment by MadoKage** [comment]
Not only are the FB Alpha cores having controls issue, they have a lot of crackling in the audio.

**Comment by Cee123** [comment]
Many of the controller issues were fixed here, with controller bitmasks
https://github.com/libretro/RetroArch/issues/11217

But for the FB Alpha cores, they weren't even working at that point so they didn't get fixed.

This latest version has a tonne of issues. I'm getting a lot of crash dumps, freezes and other bugginess with a few of the cores as well.

**Comment by Edu789** [comment]
It seems that the people who develop the Retroarch for the Wii do not test, just compile...

**Comment by SPILAB** [comment]
Hello everyone, If that can help a bit, I tested this "The current stable version is: 1.9.1" version:
- CPS2 works (with gamecube controller)
- CPS3 is here, but crash with "Esception (DSI) occurred".

**Comment by SPILAB** [comment]
Hello @Edu789 . That should be super hard to test each core on each platform.

**Comment by Edu789** [comment]
> Hello @Edu789 . That should be super hard to test each core on each platform.

Yes, but the dev that works on the PS2 version of Retroarch tests everyone he compiles ...

**Comment by Edu789** [comment]
> Hello everyone, If that can help a bit, I tested this "The current stable version is: 1.9.1" version:
> 
> * CPS2 works (with gamecube controller)
> * CPS3 is here, but crash with "Esception (DSI) occurred".

CPS 2 also does not work ...

**Comment by DjDiabolik** [comment]
Also FCEUMM it's crash........ i have tested last nightly and current stable.
When i try to run a files....... my WII goes in dump screen.
Click a button it's say Reboot but nothings appends... i need to reset manually the wii.
